### ISSUE TITLE
Make docker image to use in Jenkins pipeline

### ISSUE THREAD

**Issue by justaddcoffee** [Knowledge-Graph-Hub/kg-covid-19#77]
Continuing on from #30, we should probably make a Docker image with python3.7 and use that for the Jenkins pipeline

**Comment by justaddcoffee** [comment]
@kltm notes in #30 that python3.7 (and therefore our working pipeline) could go away at any time. A reliable Jenkins pipeline therefore depends on this ticket.

**Comment by justaddcoffee** [comment]
Possibly bump this up in priority to insulate ourselves from network issues

**Comment by justaddcoffee** [comment]
Per @kltm, this [Jenkinsfile](https://github.com/geneontology/pipeline/blob/master/Jenkinsfile) shows how we can use a docker image in Jenkins:
```
	    agent {
		docker {
		    image 'obolibrary/odkfull:v1.2.22'
		    // Reset Jenkins Docker agent default to original
		    // root.
		    args '-u root:root'
		}
	    }
```

**Comment by justaddcoffee** [comment]
Closed by #419
